src/controller/CHIPDeviceController.cpp

```diff
@@ -2372,6 +2374,32 @@ CHIP_ERROR DeviceCommissioner::ParseICDInfo(ReadCommissioningInfo & info)
         }
     }
 
+    if (!isICD)
+    {
+        return CHIP_NO_ERROR;
```

Comment:
Again, if not isICD, info.icd.idleModeDuration should be set to a corresponding sentinel value.  Nothing guarantees that the passed-in struct has not been used before, so you can't rely on its constructor initialization.

Same for activeModeDuration and activeModeThreshold.

Reply:
agree, updated, thanks